Makefile

```diff
@@ -206,7 +206,7 @@ CONTROLLER_GEN_OLD ?= $(LOCALBINOLD)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 
 ## Tool Versions
-KUSTOMIZE_VERSION ?= v3.8.7
+KUSTOMIZE_VERSION ?= v4.5.5
```

Comment:
I ran a diff between the output. The only difference was that description was added